README.md

```diff
@@ -21,6 +21,24 @@ npm run dev
 
 Check out https://abelljs.org for complete documentation.
 
+## 🚀 &nbsp; Usage
```

Comment:
Maybe "Example" or "Hello World" is a better headline here. lmk what you think

Reply:
`Example` sounds good to me.